js/sparkswap.js

```diff
@@ -271,7 +271,94 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
-        return this.privateGetV1OrderId ({ id });
+        const CONSTANTS = {
+            'STATUSES': {
+                'ACTIVE': 'ACTIVE',
+                'CANCELLED': 'CANCELLED',
+                'COMPLETED': 'COMPLETED',
+                'FAILED': 'FAILED',
+                'NONE': 'NONE',
+            },
+            'SIDES': {
+                'ASK': 'ASK',
+                'BID': 'BID',
+                'BUY': 'buy',
+                'SELL': 'sell',
+                'NONE': 'none',
```

Comment:
Is none an expected response?

Reply:
@martineehrlich nope. this is an oversight. This should be `undefined`. Info is here: https://github.com/ccxt/ccxt/wiki/Manual#trade-structure